aten/src/ATen/cuda/CUDABlas.cpp

```diff
@@ -498,6 +614,69 @@ void gemv<at::BFloat16>(CUDABLAS_GEMV_ARGTYPES(at::BFloat16)) {
 }
 #endif
 
+#define GER_CHECK_ARGVALUES(Dtype)           \
+  do {                                        \
+    CUDABLAS_NONNEGINT_CHECK(gemv<Dtype>, m); \
+    CUDABLAS_NONNEGINT_CHECK(gemv<Dtype>, n); \
+    CUDABLAS_POSINT_CHECK(gemv<Dtype>, lda);  \
+    CUDABLAS_POSINT_CHECK(gemv<Dtype>, incx); \
+    CUDABLAS_POSINT_CHECK(gemv<Dtype>, incy); \
+  } while (0)
+
+template <>
+void ger<float>(CUDABLAS_GER_ARGTYPES(float)) {
+  NoTF32Guard disable_tf32;
+  // See Note [Writing Nondeterministic Operations]
+  globalContext().alertCuBLASConfigNotDeterministic();
```

Comment:
@ptrblck Does the cuBLAS's non-determinism apply to `ger`?

Reply:
ger was already migrated so removed related changes